### ISSUE TITLE
Bug with Bayesian Independent Samples T-Test

### ISSUE THREAD

**Issue by donovaly** [jasp-stats/jasp-issues#765]
* JASP version: 0.12.2
* OS name and version: Win 10
* Analysis: Bayesiant T-Test

#### Steps to reproduce:
1. Open this JASP file: 
[Singers-bug.zip](https://github.com/jasp-stats/jasp-issues/files/4685763/Singers-bug.zip)
(Github doesn't allow to attach the file directly, thus I ZIPped it.
The Data is just the file "singers.csv" that is part of JASP.)

These are my settings for the T-Test:
![JASP_1ka1xkdMcU](https://user-images.githubusercontent.com/1828501/82963418-2071f600-9fc3-11ea-832b-ee6fc0ef275d.png)

2. Now change the option to "Group 1 < Group 2"

result: 
```
Error in private$jaspObject$addFootnoteHelper(message, symbol, colNames, : Expecting a single string value: [type=character; extent=2].

Stack trace
tryCatchOne(expr, names, parentenv, handlers[[1]])

doTryCatch(return(expr), name, parentenv, handler)

withCallingHandlers(expr = analysis(jaspResults = jaspResults, dataset = dataset, options = options), error = .addStackTrace)

analysis(jaspResults = jaspResults, dataset = dataset, options = options)

.ttestBayesianRunAnalysis(jaspResults, dataset, options, 'independent')

.ttestBayesian(jaspResults, dataset, options, errors, analysis)

.ttestBISTTest(ttestContainer, dataset, options, derivedOptions, errors, ttestState)

.ttestBayesianSetFootnotesMainTable(ttestTable, ttestResults, dependents[alreadyComputed])

ttestTable$addFootnote(message = ttestResults[['footnotes']][[var]], symbol = '', rowNames = var, colNames = 'error')

private$jaspObject$addFootnoteHelper(message, symbol, colNames, rowNames)
```

**Note:**: This bug does not occur every time. It appears for me only once after I started JASP. Maybe the stacktrace tells you more about what is actually the issue.

**Comment by Kucharssim** [comment]
Hello @donovaly,

thank you for the report. 

I am unable to reproduce this issue in JASP 13.1 or in the upcoming next version of JASP. Please reopen the issue if the problem persists.

**Comment by donovaly** [comment]
> I am unable to reproduce this issue in JASP 13.1

it is also for me fixed now with Jasp 0.13.1. Thanks.
